doc/release-notes/release-notes-0.12.0.md

```diff
@@ -166,12 +166,13 @@ creating transactions that would be replaceable under BIP 125.
 RPC: Random-cookie RPC authentication
 -------------------------------------
 
-When no `-rpcpassword` is specified, the daemon now uses a special 'cookie'
-file for authentication. This file is generated with random content when the
+The daemon now uses a special 'cookie' file for authentication.
```

Comment:
I'm not sure this is more clear. Cookie authentication is only used when no `-rpcpassword` is specified, it's not suddenly used in previous setups.